services/ui-src/src/components/fields/ChoiceListField.tsx

```diff
@@ -48,14 +48,22 @@ export const ChoiceListField = ({
   const { full_name, state, userIsAdmin } = useUser().user ?? {};
   // get form context and register field
   const form = useFormContext();
-  form.register(name);
+  const fieldIsRegistered =
+    typeof form.getValues() == "object" ? name in form.getValues() : false;
```

Comment:
Nick, why this weird typeOf line I hear you ask. Well, its because mocking form.getValues is a LOT easier and doesn't require multiple line changes if we go this route. If you disagree though or think even think this check could be written better though I'm happy to discuss it further.